src/databricksbundle/widgets/DatabricksWidgetsTest.py

```diff
@@ -86,25 +103,42 @@ def test_select_invalid_value(self):
 
         self.assertEqual("argument --myselect: invalid choice: 'my_invalid_value' (choose from 'month', 'year')", str(error.exception))
 
+    def test_select_wrong_name(self):
+        with self.assertRaises(Exception) as error:
+            self.__widgets.add_select("myselect%", ["month", "year"], "month")
+
+        self.assertEqual(
+            "The name you provided is incorrect, please provide name containing only alpha-numeric letters and _", str(error.exception)
+        )
+
+    def test_select_label_generation(self):
+        self.__widgets.add_text("myselect")
+        self.__widgets.add_text("yourselect")
+
+        self.assertEqual("01. myselect", TestingDbUtilsWidgets.get_widget("myselect").label)
+
+        self.assertEqual("02. yourselect", TestingDbUtilsWidgets.get_widget("yourselect").label)
+
     # widgets.add_multiselect() -----------------------
 
     def test_multiselect_defaults(self):
-        self.__widgets.add_multiselect("mymulti", ["January", "February", "March"], ["January", "February"])
+        self.__widgets.add_multiselect("mymulti", ["January", "February", "March"], ["January"])
         set_input()
-        self.assertEqual(["January", "February"], self.__widgets.get_value("mymulti"))
+        self.assertEqual(["January"], self.__widgets.get_value("mymulti"))
 
     def test_multiselect_defaults_overridden(self):
-        self.__widgets.add_multiselect("mymulti", ["January", "February", "March"], ["January", "February"])
+        self.__widgets.add_multiselect("mymulti", ["January", "February", "March"], ["January"])
         set_input(mymulti=["January", "March"])
         self.assertEqual(["January", "March"], self.__widgets.get_value("mymulti"))
 
     def test_multiselect_defaults_empty(self):
```

Comment:
multiselect with no default values cannot be created?

Reply:
I was not able to figure it out how to do it, therefore I believe not